logic.py

```diff
@@ -24,4 +29,8 @@ def SQLQuery(statment):
     cursor.execute(statment)
     conn.commit()
     data = logic.tuple2list(cursor.fetchall()) 
-    return data
\ No newline at end of file
+    return data
+
+def SQLInsertQuery(statment):
```

Comment:
This good, this will helps us. We can reuse this anytime we insert information. We could also see function SQLQuery, even if it returns data as long as we don't do anything with it nothing happens. If we wanted to use just one sql function the SQLQuery will work for inserting.

Reply:
Yes we can use SQLQuery function as well but there will be extra overhead of cursor.fetchall() which we don't want for insert operation.